src/layout/tooltip/index.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+export default function Tooltip() {
+
+    const style = {
+        position: "absolute",
+        left: "0px",
+        top: "0px",
+        visibility: "hidden",
+        padding: "2px",
+        "pointer-events": "none"
+    }
+
+    return (
+        <div className="tooltip" style={style}></div>
```

Comment:
```suggestion
        <div className="tooltip" style={style}>
            { props.children }
        </div>
```

We'd want to make sure that if anyone wants to use the tooltip that it can have child elements